week2/homework/src/todo.js

```diff
@@ -0,0 +1,76 @@
+const fs = require('fs');
+
+const loadJSON = () => {
+  try {
+    const dataBuffer = fs.readFileSync('todos.json');
+    const dataJSON = dataBuffer.toString();
+    return JSON.parse(dataJSON);
+  } catch (e) {
+    return [];
+  }
+};
+
+const saveJSON = todos => {
+  const dataJSON = JSON.stringify(todos);
+  fs.writeFileSync('todos.json', dataJSON);
+};
+
+const createOrderValue = array => {
+  array.forEach((elem, index) => {
+    return (elem.order = index + 1);

```

Comment:
where is this return value used?

Reply:
I think no need to use return value. Thank you